.github/workflows/build.yml

```diff
@@ -145,6 +145,8 @@ jobs:
         cmake --version | grep 3.10.3
     - name: Build
       run: script/test package
+    - name: Setup tmate session
+      uses: mxschmitt/action-tmate@v3
```

Comment:
I don't think we should add this step as part of CI.

Reply:
Was doing some debugging. Have removed it.